extras/liquibase-unlocker/pom.xml

```diff
@@ -52,14 +45,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Here instead, since we want to have a logging implementation in the final uber jar, you can leave them here